backend/src/controller/user-controller.js

```diff
@@ -223,22 +239,50 @@ module.exports.changeThePassword = async (req,res) => {
 }
 
 
-module.exports.listOfUsers = async (req,res)=> {
+module.exports.listOfUsersWithBalance = async (req,res) => {
```

Comment:
Please change this query to show only the users who is logged balance , not all the users balance , that is a major flaw

Reply:
This entire API needs some work